provision-contest/ansible/roles/keepalived/templates/keepalived.conf.j2

```diff
@@ -1,6 +1,6 @@
 vrrp_instance lb_ipv4 {
 	state MASTER
-	interface {{ ansible_facts['default_ipv4']['interface'] }}
+	interface {{ KEEPALIVED_INTERFACE|default(ansible_facts['default_ipv4']['interface']) }}
```

Comment:
This is based on what I did right? IMHO makes sense to cherry pick this if we don't have it in main yet

Reply:
Agree, will do the cherry-picking and rebasing tomorrow.